prow/jobs/test-infra/run-terraform-config.yaml

```diff
@@ -0,0 +1,142 @@
+          - image: "eu.gcr.io/kyma-project/test-infra/terraform:PR-6790"
+            securityContext:
+              privileged: true
+            workingDir: /home/prow/go/src/github.com/kyma-project/test-infra/configs/terraform/secrets-leaks-log-scanner
+            command:
+              - "/tfcmt/tfcmt"
+            args:
+              - "-owner"
+              - "$(REPO_OWNER)"
+              - "-repo"
+              - "$(REPO_NAME)"
+              - "-pr"
+              - "$(PULL_NUMBER)"
+              - "plan"
+              - "--"
+              - "terraform"
+              - "plan"
+            resources:
+              requests:
+                memory: 3Gi
+                cpu: 2
+        tolerations:
+          - key: dedicated
+            operator: Equal
+            value: high-cpu
+            effect: NoSchedule
+        nodeSelector:
+            dedicated: "high-cpu"
+    - name: pull-hide-tf-github-comments
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-hide-tf-github-comments"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-bot-github-token: "true"
+      run_if_changed: '^configs/terraform'
+      optional: true
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/terraform:PR-6790"
+            securityContext:
+              privileged: true
+            command:
+              - "/ghcomment/github-comment"
+            args:
+              - "hide"
+              - "--org"
+              - "$(REPO_OWNER)"
+              - "--repo"
+              - "$(REPO_NAME)"
+              - "-pr"
+              - "$(PULL_NUMBER)"
+            resources:
+              requests:
+                memory: 3Gi
+                cpu: 2
+        tolerations:
+          - key: dedicated
+            operator: Equal
+            value: high-cpu
+            effect: NoSchedule
+        nodeSelector:
+            dedicated: "high-cpu"
+  
+postsubmits: # runs on main
+  kyma-project/test-infra:
+    - name: post-secret-leaks-log-scanner-tf-apply
+      annotations:
+        pipeline.trigger: "pr-merge"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "post-secret-leaks-log-scanner-tf-apply"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      run_if_changed: '^configs/terraform/secrets-leaks-log-scanner'
+      optional: true
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/terraform:PR-6790"
+            securityContext:
+              privileged: true
```

Comment:
Is privileged: true required in this case?

Reply:
no